### PR TITLE
Add powerpc builds

### PR DESCRIPTION
**Pull request by brson**
Completely untested. How does this look @alexcrichton?

cc #531 @lu-zero

### CODE REVIEW

**Comment by alexcrichton** [comment]
Ah looks like the build for OpenSSL needs to be updated to build powerpc as well, but otherwise I think this should be all that's needed.

**Comment by brson** [comment]
Updated the openssl builds.

**Comment by alexcrichton** [comment]
Looks like the OS may be `linux-ppc` instead of `linux-powerpc` (and similar for others)

**Comment by brson** [comment]
Updated again.

File: ci/run-docker.sh [hunk 1]
```diff
@@ -64,6 +64,21 @@ case $TARGET in
...
     OPENSSL_AR=x86_64-unknown-netbsd-ar
     ;;
+  powerpc-*-linux-*)
+    OPENSSL_OS=linux-ppc
+    OPENSSL_CC=powerpc-linux-gnu-gcc
+    OPENSSL_AR=powerpc-linux-gnu-ar
+    ;;
+  powerpc64-*-linux-*)
+    OPENSSL_OS=linux-ppc64
+    OPENSSL_CC=powerpc64-linux-gnu-gcc
```
Comment (alexcrichton) [inline ci/run-docker.sh]: Oh the failure now is:

```
powerpc64-linux-gnu-gcc: Command not found
```

I think that powerpc64 works by using the `powerpc-linux-gnu` compiler and then passing `-m64` as a flag

**Comment by brson** [comment]
Updated again.

**Comment by alexcrichton** [comment]
Hey look it works!

**Comment by lu-zero** [comment]
Great !

**Pull request merged**